tb-dev-challenge/MenuGroups+CoreDataClass.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MenuGroups+CoreDataClass.swift
+//  tb-dev-challenge
+//
+//  Created by Bhavin Kapadia on 2022-03-22.
+//
+//
+
+import UIKit
+import CoreData
+
+@objc(MenuGroups)
+public class MenuGroups: NSManagedObject {
```

Comment:
did you create this to access all the menu items? `public` means it's even accessible outside the project, say we have a SDK and wanna expose certain methods/interfaces to the SDK users then we can make them public. Also it might be redundant when there's a `ofMenuItem` already

Reply:
yea i guess i should make it private but now sure how that works since it causes a build issues, might need to pair to understand this fix.

Reply:
it can just be default, which is `internal`

Reply:
I autogenerated the NSMangedObject sub class using xcode's editor. I was able to remove the `public` though.